samples/core/visualization/markdown.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp.dsl as dsl
+import kfp.components as comp
+
+# Advanced function
+# Demonstrates imports, helper functions and multiple outputs
+from typing import NamedTuple
+
+
+def markdown_visualization() -> NamedTuple('VisualizationOutput', [('mlpipeline_ui_metadata', 'UI_metadata')]):
+    import json
+
+    # Exports a sample tensorboard:
+    metadata = {
+        'outputs': [
+            {
+                # Markdown that is hardcoded inline
+                'storage': 'inline',
+                'source': '# Inline Markdown\n[A link](https://www.kubeflow.org/)',
```

Comment:
nit: use '''?
There is a syntax for multiline string in python

Reply:
That is a good point! Thank you for the idea.

I have changed it to use multiline string, but note that each new line should start without any space prefix, otherwise markdown viewer will have problem understanding the title/bullet point symbols.